### ISSUE TITLE
Tests for negative lengths for arraybuffer and typed array constructors don't match current spec draft

### ISSUE THREAD

**Issue by bzbarsky** [w3c/web-platform-tests#1579]
typedarrays/test_constructors.html has:

```
[-0xF1000000, 0xF000000]
```

However per spec for a typed array http://people.mozilla.org/~jorendorff/es6-draft.html#sec-%typedarray%-length says:

```
Let numberLength be ToNumber(length).
Let elementLength be ToLength(numberLength).
```

and http://people.mozilla.org/~jorendorff/es6-draft.html#sec-tolength will convert negative numbers to +0.  So the 0xF0000000 should be 0.

Similar for typedarrays/ArrayBuffer_constructor.html

**Comment by annevk** [comment]
This was fixed in #2084, no?

**Comment by bzbarsky** [comment]
Looks like it, yes.  That PR does explicitly reference this issue...
